rpc-server/src/modules/blocks/mod.rs

```diff
@@ -7,4 +7,5 @@ pub struct CacheBlock {
     pub block_height: near_primitives::types::BlockHeight,
     pub block_timestamp: u64,
     pub latest_protocol_version: near_primitives::types::ProtocolVersion,
+    pub chunks_count: u64,
```

Comment:
in `nearcore` the similar field is named `chunks_included`, please use that name instead